securedrop/journalist_templates/admin.html

```diff
@@ -2,57 +2,67 @@
 {% block body %}
 <h1>{{ gettext('Admin Interface') }}</h1>
 
+<section aria-labelledby="add-user-heading">
+<h2 id="add-user-heading" class="visually-hidden">Add User</h2>
 <form action="{{ url_for('admin.add_user') }}">
-  <button type="submit" id="add-user">
+  <button type="submit" id="add-user" aria-label="{{ gettext('Add User') }}">
+    {# FIXME:
     <img src="{{ url_for('static', filename='icons/plus.png') }}" class="icon" width="13" height="15" alt="">
+    #}
     {{ gettext('ADD USER') }}
   </button>
 </form>
+</section>
```

Comment:
Great catch; thank you.  I'm not sure why 54834e22309e3c8dfaead2a07a650a9af87e7ca1 added this `button` rather than an `a.btn`, which is what 2be57f94ea50683c5a7a3d63c67f25a3097219bf subsequently did for the **Instance Config** button.  Fixed in f4d25065d6d3a74bbbbae7e264f91173389de3f9.